scripts/build.sh

```diff
@@ -12,16 +12,25 @@ cd "${CONSTRUCT_ROOT}"
 # Constructor should be latest for non-native building
 # See https://github.com/conda/constructor
 echo "***** Install constructor *****"
-conda install -y "constructor>=3.3.1" jinja2 curl libarchive -c conda-forge --override-channels
+conda install --yes jinja2 curl libarchive --channel conda-forge --override-channels
```

Comment:
You can install constructor here and override it with pip later on so that you don't have to manually install the dependencies of constructor.

Reply:
I'm personally always confused about how that plays out. There are also some new nsis dependencies.

In either case the errors seem to be unrelated to the removal of this

Reply:
How about we carry that patch (the one line with `used_repodata.pop("_mod", None)`) in constructor-feedstock?

Reply:
ok, let me try to fork and apply that patch only, and see what happens.

Reply:
I guess i'm ok with debugging constructor, but this isn't any worse than before. 

Can we move forward with: https://github.com/conda-forge/miniforge/pull/362

Reply:
What's the point of releasing something that doesn't work?

Reply:
It works just as well as the other things used to.

The problem is just as well hidden as it always has been.